jpa/eclipselink.jpa.test.jse/src/org/eclipse/persistence/jpa/test/basic/TestEntityManagerInterface.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 1998, 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1998, 2019 IBM Corporation and/or its affiliates. All rights reserved.
```

Comment:
no need to have '1998' here, '2019' is enough

Reply:
Noted, and fixed.  Thanks!